middleware.go

```diff
@@ -150,7 +155,7 @@ func (h *handler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	context, err := h.Auth.Validate(authToken)
 	if err != nil {
 		//ToDo: How to handle logging, printing to stdout isn't the best thing
-		fmt.Println("Failed to validate token. ", err)
+		Log("Failed to validate token. ", err)
```

Comment:
Sorry missed this when looking it previously but now that we are using `Printf` instead of `Println` this needs to be changed to `Log("Failed to validate token: %s", err)`